### PR TITLE
Add ForeignLanguage entities

### PR DESCRIPTION
**Pull request by ArtyomBanchikov**

### CODE REVIEW

**Review by VideoRab — APPROVED** [review]

File: CV/CV.DAL/Entities/ForeignLanguage/LanguageEverydayTopicEntity.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using CV.DAL.Infraestructure;
+
+namespace CV.DAL.Entities.ForeignLanguage
+{
+    public class LanguageEverydayTopicEntity : HasId
+    {
+#nullable disable
```
Comment (Blackolf931) [inline CV/CV.DAL/Entities/ForeignLanguage/LanguageEverydayTopicEntity.cs]: What? ???????

File: CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using CV.DAL.Infraestructure;
+
+namespace CV.DAL.Entities.ForeignLanguage
+{
+    public class LanguageProfessionalTopicEntity:HasId
+    {
+#nullable disable
+#nullable enable
```
Comment (eeveesharp) [inline CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs]: ?

File: CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿using CV.DAL.Infraestructure;
+
+namespace CV.DAL.Entities.ForeignLanguage
+{
+    public class LanguageProfessionalTopicEntity:HasId
+    {
+#nullable disable
```
Comment (Blackolf931) [inline CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs]: The same ???????

**Review by Blackolf931 — COMMENTED** [review]
Why you did it in one PR?

File: CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs (hunk 2 above)
Comment (ArtyomBanchikov) [reply CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs]: There will be 3 links for Reading, Writing and Speaking level

File: CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs (hunk 3 above)
Comment (ArtyomBanchikov) [reply CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs]: There will be 3 links for Reading, Writing and Speaking level

File: CV/CV.DAL/Entities/ForeignLanguage/LanguageEverydayTopicEntity.cs (hunk 1 above)
Comment (ArtyomBanchikov) [reply CV/CV.DAL/Entities/ForeignLanguage/LanguageEverydayTopicEntity.cs]: There will be 3 links for Reading, Writing and Speaking level

File: CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs (hunk 2 above)
Comment (Blackolf931) [reply CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs]: It will be, but i don't see them

Comment (ArtyomBanchikov) [reply CV/CV.DAL/Entities/ForeignLanguage/LanguageProfessionalTopicEntity.cs]: You said that we don't have to make connections yet

**Review by Blackolf931 — APPROVED** [review]

**Review by Blackolf931 — COMMENTED** [review]
What happend with build?

**Comment by ArtyomBanchikov** [comment]
Someone add repository to SkillEntity, but I changed folder for this entities in this PR

**Review by eeveesharp — APPROVED** [review]

**Pull request merged**